pennylane/interfaces/tf.py

```diff
@@ -12,190 +12,8 @@
-
-For example:
-
-.. code-block:: python
-
-    dev = qml.device('default.qubit', wires=2)
-
-    @qml.qnode(dev, interface='tf')
-    def circuit(phi, theta):
-        qml.RX(phi[0], wires=0)
-        qml.RY(phi[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.PhaseShift(theta, wires=0)
-        return qml.expval(qml.PauliZ(0))
-
-    phi = Variable([0.5, 0.1])
-    theta = Variable(0.2)
-
-    with tf.GradientTape() as tape:
-        # Use the circuit to calculate the loss value
-        loss = circuit(phi, theta)
-        phi_grad, theta_grad = tape.gradient(loss, [phi, theta])
-
-Now, printing the gradients, we get:
-
->>> phi_grad
-array([-0.47942549,  0.        ])
->>> theta_grad
--5.5511151231257827e-17
-
-.. _pytf_optimize:
-
-Optimization using TensorFlow
------------------------------
-
-To optimize your hybrid classical-quantum model using the TensorFlow eager interface,
-you **must** make use of the TensorFlow optimizers provided in the ``tf.train`` module,
-or your own custom TensorFlow optimizer. **The** :ref:`PennyLane optimizers <optimization_methods>`
-**cannot be used with the TensorFlow interface, only the** :ref:`numpy_qnode`.
-
-For example, to optimize a TF-interfacing QNode (below) such that the weights ``x``
-result in an expectation value of 0.5, we can do the following:
-
-.. code-block:: python
-
-    import tensorflow as tf
-
-    # For TensorFlow 2.X:
-    from tensorflow import Variable
-
-    # if using TensorFlow 1.X, you will
-    # instead require the following three lines:
-    #
-    # import tensorflow.contrib.eager as tfe
-    # Variable = tfe.Variable
-    # tf.enable_eager_execution()
-
-    import pennylane as qml
-
-    dev = qml.device('default.qubit', wires=2)
-
-    @qml.qnode(dev, interface='tf')
-    def circuit(phi, theta):
-        qml.RX(phi[0], wires=0)
-        qml.RY(phi[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.PhaseShift(theta, wires=0)
-        return qml.expval(qml.PauliZ(0))
-
-    phi = Variable([0.5, 0.1], dtype=tf.float64)
-    theta = Variable(0.2, dtype=tf.float64)
-
-    opt = tf.train.GradientDescentOptimizer(learning_rate=0.1)
-    steps = 200
-
-    for i in range(steps):
-        with tf.GradientTape() as tape:
-            loss = tf.abs(circuit(phi, theta) - 0.5) ** 2
-            grads = tape.gradient(loss, [phi, theta])
-
-        opt.apply_gradients(zip(grads, [phi, theta]), global_step=tf.train.get_or_create_global_step())
-
-
-The final weights and circuit value are:
-
->>> phi
-<tf.Variable 'Variable:0' shape=(2,) dtype=float64, numpy=array([ 1.04719755,  0.1       ])>
->>> theta
-<tf.Variable 'Variable:0' shape=() dtype=float64, numpy=0.20000000000000001>
->>> circuit(phi, theta)
-<tf.Tensor: id=106269, shape=(), dtype=float64, numpy=0.5000000000000091>
-
-
-Code details
-^^^^^^^^^^^^
+This module contains the :func:`TFEQNode` function to convert Numpy-interfacing quantum nodes to TensorFlow
+Eager mode compatible quantum nodes.
```

Comment:
This file is outdated

Reply:
That's linked to the TFE interface tutorial being outdated, right?

Reply:
Yep, see other comment :slightly_smiling_face: